### PR TITLE
[appveyor] Fix failure reporting

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #17464 |
| License | MIT |
| Doc PR | - |

Status: needs work

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ install:
     - if [[ $deps != skip ]]; then COMPONENTS=$(find src/Symfony -mindepth 3 -type f -name phpunit.xml.dist -printf '%h\n'); fi;
     - if [[ $deps != skip && $deps ]]; then php .travis.php $TRAVIS_COMMIT_RANGE $TRAVIS_BRANCH $COMPONENTS; fi;
     - if [[ $deps = high && $TRAVIS_BRANCH = master ]]; then SYMFONY_VERSION=$(git ls-remote --heads | grep -o '/[1-9].*' | tail -n 1 | sed s/.//); else SYMFONY_VERSION=$(cat composer.json | grep '^ *"dev-master". *"[1-9]' | grep -o '[0-9.]*'); fi;
-    - if [[ $deps = high && $TRAVIS_BRANCH = master ]]; then git fetch origin $SYMFONY_VERSION; git checkout -m FETCH_HEAD; fi;
+    - if [[ $deps = high && $TRAVIS_BRANCH = master ]]; then git fetch origin $SYMFONY_VERSION; git checkout -m FETCH_HEAD; COMPONENTS=$(find src/Symfony -mindepth 3 -type f -name phpunit.xml.dist -printf '%h\n'); fi;
```
Comment (nicolas-grekas) [inline .travis.yml]: just syncing with master here

**Comment by nicolas-grekas** [comment]
Status: needs review
PR is ready, and shows that we have a broken 2.3...

**Comment by nicolas-grekas** [comment]
PR now embeds fixes for our lowest supported PHP version and for Windows.
Green.

**Comment by dunglas** [comment]
:+1: (but I'm not confortable at all with Regedit.exe so I blindly trust you).

**Pull request merged**